src/index.html

```diff
@@ -1,12 +1,464 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+
+    <header class="header">
+      <a href="#home">
+        <img
+        src="images/logo.png"
+        alt="moyo"
+        class="logo header__logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="."
+              class="nav__link nav__link--is-active is-active"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="." class="nav__link">SAMSUNG</a>
```

Comment:
Add corresponding href according to the content for each link
```suggestion
            <a href="#samsung" class="nav__link">SAMSUNG</a>
```